src/components/TermsDialog.tsx

```diff
@@ -0,0 +1,117 @@
+import { Dialog, Transition } from '@headlessui/react';
+import clsx from 'clsx';
+import { Fragment, useState } from 'react';
+import { termsAgreedAtom } from 'store/app';
+import { useSetAtom } from 'jotai';
+
+// Increment every time the current terms change.
+export const currentTerms = 0;
```

Comment:
without this it reads like it contains the term data itself. consider `currentTermsIndex`

Reply:
done